bullet_train-super_scaffolding/lib/scaffolding/attribute.rb

```diff
@@ -58,6 +58,20 @@ def association_class_name
     name.split("_id").first
   end
 
+  def association_name
+    association_class_name.tableize
```

Comment:
`tableize` will also pluralize, which I don't know if we want for `belongs_to`/`has_one` associations. For those, we'd probably want `underscore.tr("/", "_")`.

Reply:
🤔 Hmm, good point. Currently I think this method is only being called in places where we have a `has_many` or `has_many through:` relationship, but I can see how this would be confusing and potentially lead to problems down the road. I'll clean it up a bit.